.appveyor.yml

```diff
@@ -46,7 +46,7 @@ install:
   - set PYTHONPATH=C:\Python27
   - set PATH=%PYTHONPATH%;%PYTHONPATH%\Scripts;%PATH%
   - python --version
-  - pip install jinja2 nose nose-timer
+  - pip install jinja2 nose nose-timer PyYAML
```

Comment:
The windows vagrant box will need the `PyYAML` package too in order to be able to run these tests.